Documentation/planned-migration-and-dr.md

```diff
@@ -0,0 +1,13 @@
+---
+title: DR Overview
```

Comment:
```suggestion
title: Disaster Recovery Overview
```

Reply:
Thanks for the suggestions! Updated the PR accordingly.